src/server.h

```diff
@@ -1281,6 +1281,7 @@ struct redisServer {
     client *current_client;     /* Current client executing the command. */
     rax *clients_timeout_table; /* Radix tree for blocked clients timeouts. */
     long fixed_time_expire;     /* If > 0, expire keys against server.mstime. */
+    long stop_lazy_expire;      /* If > 0, don't trigger lazy expire */
```

Comment:
maybe use `suspend` instead of `stop`?
or maybe it should be placed next to `active_expire_enabled` and named similarly (`lazy_expire_disabled`)?